src/components/bookings/SeatingGrid.vue

```diff
@@ -26,6 +26,13 @@ const rowNumbers = computed(() => {
     .sort(function (a, b) {
       return +a - +b;
     });
+  // return allSeats.value
+  //   .map((str) => {
+  //     if (str.includes("A")) str.substring(1);
+  //   })
+  //   .sort(function (a, b) {
+  //     return +a - +b;
+  //   });
```

Comment:
If it is commented out it should be removed unless there is a good reason to keep it 👮 🚔